fuzzy/docker/level1/Dockerfile.libmath

```diff
@@ -0,0 +1,17 @@
+FROM verificarlo/verificarlo:latest
```

Comment:
Can you instead build using the verificarlo container that we build here? I believe the name is `gkiar/fuzzy:ubuntu14.04-base`